desc/objectives/linear_objectives.py

```diff
@@ -177,8 +177,7 @@ def compute(self, *args, **kwargs):
             x = kwargs.get(self.args[0], args[0])
         else:
             x = kwargs.get(self.args[0])
-        Rb = jnp.dot(self._A, x)
-        return self._shift_scale(Rb)
+        return jnp.dot(self._A, x)
```

Comment:
Does it make any sense to impement bounds on FixBoundaryR and Z? Like if you want to let a mode vary only a bit?

Reply:
The bounds do get applied to linear objectives, just like every other Objective function. The bounds are implemented in `_Objective.compute_scaled`, which internally calls `_Objective.compute`. All of the objective Jacobians are computed using the scaled version (with weights, etc.), and the constraint utility function logic also uses the new scaled compute functions.

Reply:
I see that, but where in the linear objectives do you put the bounds argument I mean? In all the other objectives the bounds is an argument passed in where you specify a tuple of upper and lower bounds, instead of giving a target. This is missing in all of the linear objectives. Maybe I am misunderstanding your point here

Reply:
Oh good catch! I just added them.